### DISCUSSION TITLE
ONNX support for Stan models

### DISCUSSION THREAD

**Discussion by bparbhu** [microsoft/onnxruntime#11575] [General]
Hey all,

I've been thinking more about how to improve interoperability for Stan and how Stan users
can take advantage of additional performance gains by possibly switching over stan models to be saved
in the onnx format. I'm interested in helping with this and would want to come up with a design doc around this.
Any feedback on this is much appreciated and always happy to help.

Thanks again,

-Brian

**Comment by tianleiwu** [comment]
First, it need a tool to convert Stan models to ONNX model. For example, add the support in https://github.com/onnx/onnxmltools.

Once there is model, it is possible to do a performance evaluation using OnnxRuntime.

**Reply by bparbhu** [reply]
Cool, thanks again, I'll make a post there as well.
